src/components/CommunityEvents.tsx

```diff
@@ -0,0 +1,256 @@
+      Join Discord
+    </ButtonLink>
+  )
+}
+
+const CommunityEvents = () => {
+  const { language } = useI18next()
+  const { t } = useTranslation()
+  const [state, setState] = useState<State>({
+    pastEventData: [],
+    upcomingEventData: [],
+    loading: true,
+    hasError: false,
+  })
+
+  useEffect(() => {
+    try {
+      const fetchCalendarData = async () => {
+        const events = await getData(`${GATSBY_FUNCTIONS_PATH}/calendarEvents`)
+        const pastEventData = events.pastEvents.map((event) => {
+          return {
+            date: event.start.dateTime,
+            title: event.summary,
+            calendarLink: event.htmlLink,
+            pastEventLink: event.location,
+          }
+        })
+        const upcomingEventData = events.futureEvents.map((event) => {
+          return {
+            date: event.start.dateTime,
+            title: event.summary,
+            calendarLink: event.htmlLink,
+            pastEventLink: event.location,
+          }
+        })
+        setState({
+          ...state,
+          pastEventData,
+          upcomingEventData,
+          loading: false,
+          hasError: false,
+        })
+      }
+      fetchCalendarData()
+    } catch {
+      setState({ ...state, loading: false, hasError: true })
+    }
+  }, [])
+
+  const renderEventDateTime = (
+    date,
+    language,
+    params = {
+      year: "numeric",
+      month: "long",
+      day: "numeric",
+      hour12: false,
+      hour: "numeric",
+      minute: "numeric",
+    }
+  ) => {
+    return DateTime.fromISO(date).setLocale(language).toLocaleString(params)
+  }
+
+  const renderEventLink = (link, title) => {
+    return (
+      <Link to={link} fontWeight="700">
+        {title}
+      </Link>
+    )
+  }
+
+  const renderEvent = (event, language) => {
+    const { date, title, calendarLink } = event
+    const params = {
+      year: "numeric",
+      month: "short",
+      day: "numeric",
+    }
+    return (
+      <Flex gap={6}>
+        <Text>{renderEventDateTime(date, language, params)}</Text>
+        {renderEventLink(calendarLink, title)}
+      </Flex>
+    )
+  }
+
+  return (
+    <Flex w="full" flexDirection={{ base: "column", lg: "row" }}>
+      <Center w={{ base: "100%", lg: "40%" }} px={16}>
+        <Box>
+          <Heading>
+            <Translation id="community-events-content-heading" />
+          </Heading>
+          <Text>
+            <Translation id="community-events-content-1" />
+          </Text>
+          <Text>
+            <Translation id="community-events-content-2" />
+          </Text>
+        </Box>
+      </Center>
+      <Flex
+        w={{ base: "100%", lg: "60%" }}
+        flexDirection={{ base: "column", lg: "row" }}
+      >
+        <Flex
+          w={{ base: "100%", lg: "50%" }}
+          bg="layer2Gradient"
+          p={12}
+          textAlign="center"
+          flexDir="column"
+        >
+          <Text fontSize="md" fontWeight="bold">
```

Comment:
16px gap on all screen sizes

<img width="426" alt="Screenshot 2023-06-08 at 12 35 15" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/fdcfdf52-779e-4995-b349-5417eb4dd780">


```suggestion
          <Text mb={1} fontSize="md" fontWeight="bold">
```